build/lib/client/client_request.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+from backend_http.do_request import (
```

Comment:
Yo llamaría al módulo simplemente `backend.requests`, pero cuestión de gustos 😛.

Reply:
I'd prefer to maintain it as is for now to avoid any conflict with the actual `requests` library. Is that fair enough?